tvm-opcodes/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id("tsa.kotlin-conventions")
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.9.22"
+}
+
+dependencies {
+    implementation(project(":tvm-disasm"))
+
+    implementation("org.ton:ton-kotlin-tvm:0.3.1")
+    implementation("org.ton:ton-kotlin-hashmap-tlb:0.3.1")
```

Comment:
Are they really required?

Reply:
Yes, because we use them to parse dict with methods

Reply:
Well, I was able to remove `"org.ton:ton-kotlin-hashmap-tlb:0.3.1"` from `tvm-opcodes`, but since it depends on `tvm-disasm`, it won't make much difference